$NetBSD$

(1) convert info directory
(2) Add ${PREFIX}/share/emacs/site-lisp for magit.el when
    byte compile

--- Makefile.orig	2013-11-07 09:19:23.000000000 +0900
+++ Makefile	2013-11-07 11:59:12.000000000 +0900
@@ -1,7 +1,7 @@
 PREFIX  ?= /usr/local
 datarootdir ?= $(PREFIX)/share
 lispdir ?= $(datarootdir)/emacs/site-lisp/magit
-infodir ?= $(datarootdir)/info
+infodir ?= $(PREFIX)/info
 docdir  ?= $(datarootdir)/doc/magit
 execdir ?= $(PREFIX)/bin
 
@@ -27,7 +27,7 @@ INSTALL_INFO ?= install-info
 
 EFLAGS ?= -L ../git-modes -L ../cl-lib
 EMACS  ?= emacs
-BATCH   = $(EMACS) $(EFLAGS) -batch -Q -L .
+BATCH   = $(EMACS) $(EFLAGS) -batch -Q -L . -L @@BUILDLINK_DIR@@/share/emacs/site-lisp/git-modes
 BATCHE  = $(BATCH) -eval
 BATCHC  = $(BATCH) -f batch-byte-compile
 
